yarn.lock

```diff
@@ -1977,7 +1977,7 @@
   dependencies:
     "@types/glob" "*"
 
-"@types/aws-lambda@^8.10.110":
+"@types/aws-lambda@^8.10.109", "@types/aws-lambda@^8.10.110":
   version "8.10.110"
```

Comment:
Do you need this specific of aws-lambda? If not please revert all changes to `yarn.lock`

Reply:
Good spot, done.